config/application.rb

```diff
@@ -44,5 +44,7 @@ class Application < Rails::Application
 
     # This makes sure our Postgres enums function are persisted to the schema
     config.active_record.schema_format = :sql
+
+    config.logger = ActiveSupport::TaggedLogging.new(Logger.new(STDOUT)) if ENV['RAILS_LOG_TO_STDOUT'].present?
```

Comment:
This duplicates line 67 of config/environments/production.rb. I don't think we need it.  I think this ticket should just be a Puppet change.

Reply:
I was curious about that. I'll close this.